### ISSUE TITLE
Should have confirm to delete dial

### ISSUE THREAD

**Issue by jimk60046** [sblask/webextension-bookmark-dial#22]
### New Issue Checklist (tick off like this: `[x]`)

- [x ] I checked existing issues (open and closed) for possible duplicates
- [x ] I can reproduce the problem on the latest stable version (not nightly!)

### What is your browser?

- [ x] Firefox
- [ ] Chrome

### What is your operating system?

- [ ] Linux
- [ ] Mac
- [x ] Windows

### Description (please include examples/screenshots where applicable)
I really like this addon better than the rest  but sometimes I accidentally delete a dial clicking the x even when I don't know it. Please add a confirm to delete from the main dials page.

**Comment by sblask** [comment]
I agree, happens to me as well every now and then. I'll see how I can do it when I find some time.

**Comment by sblask** [comment]
It took a while , but there is now a setting that enables a confirmation dialogue.
